eyelid-match-ops/src/primitives/yashe/test/hamming.rs

```diff
@@ -0,0 +1,87 @@
+#[cfg(test)]
+mod tests {
```

Comment:
I think you might be looking for a whole-module config, which is written like this:
```rust
#![cfg(test)]
```

But we usually write it in the `test.rs` module file:
```rust
#[cfg(test)]
mod hamming;
```

Reply:
done